.automation/test/mega-linter-plugin-test/test.megalinter-descriptor.yml

```diff
@@ -1,13 +1,13 @@
 descriptor_id: TEST
 descriptor_type: format
 file_extensions:
-  - ".jsonc"
+- .jsonc
+is_plugin: true
 linters:
-  # JSONLINT
-  - linter_name: jsonlint
-    linter_url: https://github.com/zaach/jsonlint
-    examples:
-      - "jsonlint myfile.jsonc"
-    install:
-      dockerfile:
-        - RUN npm install jsonlint -g && echo "This is my test MegaLinter plugin !"
+- examples:
+  - jsonlint myfile.jsonc
+  install:
+    dockerfile:
+    - RUN npm install jsonlint -g && echo "This is my test MegaLinter plugin !"
```

Comment:
This is a minor pre-existing issue, but there shouldn't be a space before the exclamation point.